web_widget_text_markdown/static/tests/js/web_widget_text_markdown.js

```diff
@@ -2,75 +2,78 @@
-                assert.expect(1);
+        function() {
+            QUnit.module("FieldTextMarkDown");
+            QUnit.test(
+                "bootstrap markdown widget are correctly rendered (preview)",
+                function(assert) {
+                    assert.expect(1);
 
-                var done = assert.async();
+                    var done = assert.async();
 
-                createAsyncView({
-                    View: FormView,
-                    model: 'partner',
-                    data: this.data,
-                    arch: this.arch,
-                    res_id: 1,
-                }).then(function (form) {
-                    _.defer(function () {
-                        assert.strictEqual(form.$('.md-editor').length, 0);
-                        form.destroy();
-                        done();
+                    createAsyncView({
+                        View: FormView,
+                        model: "partner",
+                        data: this.data,
+                        arch: this.arch,
+                        res_id: 1,
+                    }).then(function(form) {
+                        _.defer(function() {
+                            assert.strictEqual(form.$(".md-editor").length, 0);
+                            form.destroy();
+                            done();
+                        });
                     });
-                });
-            }
-        );
-        QUnit.test('bootstrap markdown widget are correctly rendered (edit)',
-            function (assert) {
-                assert.expect(1);
+                }
+            );
+            QUnit.test(
+                "bootstrap markdown widget are correctly rendered (edit)",
+                function(assert) {
+                    assert.expect(1);
 
-                var done = assert.async();
+                    var done = assert.async();
 
-                createAsyncView({
-                    View: FormView,
-                    model: 'partner',
-                    data: this.data,
-                    arch: this.arch,
-                    res_id: 1,
-                    viewOptions: {mode: 'edit'},
-                }).then(function (form) {
-                    _.defer(function () {
-                        assert.strictEqual(form.$('.md-editor').length, 1);
-                        form.destroy();
-                        done();
+                    createAsyncView({
+                        View: FormView,
+                        model: "partner",
+                        data: this.data,
+                        arch: this.arch,
+                        res_id: 1,
+                        viewOptions: {mode: "edit"},
+                    }).then(function(form) {
+                        _.defer(function() {
+                            assert.strictEqual(form.$(".md-editor").length, 1);
+                            form.destroy();
+                            done();
+                        });
```

Comment:
Example in point - I can review this better if the black/isort stuff is in a separate commit :-)

Reply:
He I did the module migration as the main commit and git add all the changes into one, I did:
`pre-commit run --all --show-diff-on-failure --verbose --color always` on my local and the command on maintainer-tools
after realizing travis cl had issues with previous pull request.

Reply:
Hm, when I do the same, I get a different result:

```
cp -R web_widget_text_markdown /tmp  # backup your work
git reset --hard HEAD~2  # reset to before your work, but after adding history
pre-commit run --all --show-diff-on-failure --verbose --color always  # pre-commit on history 
git add . && git commit -m "[UPD] black, isort, prettier"   # add this
rm -Rf web_widget_text_markdown
cp -R /tmp/web_widget_text_markdown .  # warp files to after your work
pre-commit run --all --show-diff-on-failure --verbose --color always  # pre-commit, to be sure
git diff  # seeing lots of changes
git restore web_widget_text_markdown/i18n  # why were these removed?
git add . && git commit -m "[MIG] migration to 13.0"  # the commit that defines your migration activity
git push sunflowerit
```

Result is [here](https://github.com/sunflowerit/web/tree/13.0-mig-web-widget-text-markdown)

If you look at the [topmost commit](https://github.com/sunflowerit/web/commit/f4edca56fba47455be8b070cde42319c94cb1c54), now that is something I can easily review... and have left my comments there.

Reply:
well after a long while working out how to fix it, manage to understand basic concepts though complete fixture was not
added but clean up is ok. What I tried to do was:
`git remote add sfit https://github.com/sunflowerit/web.git
git fetch sfit
git log sfit/13.0-web-widget-text-markdown
git cherry-pick f4edca5
git push origin head:13.0-web-widget-text-markdown
`